ale_linters/elixir/credo.vim

```diff
@@ -3,8 +3,8 @@
 function! ale_linters#elixir#credo#Handle(buffer, lines) abort
   " Matches patterns line the following:
   "
-  " stdin:19: F: Pipe chain should start with a raw value.
-  let l:pattern = '\v^stdin:(\d+):?(\d+)?: (.): (.+)$'
+  " lib/filename.ex:19:7: F: Pipe chain should start with a raw value.
+  let l:pattern = '\v^.+:(\d+):?(\d+)?: (.): (.+)$'
```

Comment:
This could also be `'\v:(\d+):?(\d+)?: (.): (.+)$'` (just anchor at the tail). Let me know which is preferred.